src/nft/components/bag/BagHeader.tsx

```diff
@@ -47,6 +43,26 @@ const IconWrapper = styled.button`
     },
   }) => `opacity ${duration.medium} ${timing.ease}`};
 `
+const NumberDot = styled.div`
+  align-items: center;
+  background-color: ${({ theme }) => theme.accentActive};
+  border-radius: 50%;
+  display: flex;
+  font-size: 10px;
```

Comment:
can we extract font settings to a new preset style called "utilityBadge" in src/nft/css/common.css.ts?

In Figma it seems like this is a new typography preset style: 
//styleName: Utility/Badge;
font-family: Inter;
font-size: 10px;
font-weight: 600;
line-height: 12px;
letter-spacing: 0em;
text-align: center;

https://www.figma.com/file/bCN7G6oArm7pPts03jvJ5x/Unification?node-id=17444%3A89794&t=v1DnxqDOOGLhXeum-0

Reply:
also this can be applied to the text div described in below comment

Reply:
i think this could wait until we handle Text styling in the component library, as I don't think this is particularly solidified in the design system yet